### DISCUSSION TITLE
Download Cover

### DISCUSSION THREAD

**Discussion by casahernando** [comictagger/comictagger#671] [Ideas]
Hi. I've been thinking about the possibility of downloading the correct one from comicvine when an issue doesn't have a cover, or the cover isn't correct.

Would that be very difficult?

Thank you very much

**Comment by lordwelch** [comment]
All the covers are actually already downloaded in the cache. You can use `-vv` on the cli to list the cache directory or in the latest beta it is listed in the settings window `ctrl+shift+s` if someone wants to contribute a PR I'd consider it
